static/app/views/replays/deadClickList.tsx

```diff
@@ -0,0 +1,34 @@
+import * as Layout from 'sentry/components/layouts/thirds';
+import PageFiltersContainer from 'sentry/components/organizations/pageFilters/container';
+import {PageHeadingQuestionTooltip} from 'sentry/components/pageHeadingQuestionTooltip';
+import SentryDocumentTitle from 'sentry/components/sentryDocumentTitle';
+import {t} from 'sentry/locale';
+import useOrganization from 'sentry/utils/useOrganization';
+
+export default function DeadClickList() {
+  const organization = useOrganization();
+  const hasDeadCicks = organization.features.includes(
+    'session-replay-rage-dead-selectors'
+  );
+
+  return hasDeadCicks ? (
+    <SentryDocumentTitle title={`Top Selectors with Dead Clicks — ${organization.slug}`}>
+      <Layout.Header>
+        <Layout.HeaderContent>
+          <Layout.Title>
+            {t('Top Selectors with Dead Clicks')}
+            <PageHeadingQuestionTooltip
+              title={t('See the top selectors your users have dead clicked on.')}
```

Comment:
Open for suggestions to improve this tooltip wording, but we can always change it anytime @jas-kas